src/json/webhooks/ReferralClassicDisabled.schema.json

```diff
@@ -45,6 +55,13 @@
       "description": "The Id of the program this code was created for"
     }
   },
-  "required": ["id", "referrer", "referred", "dateReferralStarted"],
+  "required": [
+    "id",
+    "referrer",
+    "referred",
+    "referralCodeUsed",
+    "shareLinkUsed",
```

Comment:
@DerekSiemens I'm not sure `shareLinkUsed` should be required here - it's possible to create referrals without a share link, i.e. from the portal. Can you please check with @slisaasquatch on this one?

Reply:
Good point, Ill check with him